src/license/mapping.xml

```diff
@@ -42,6 +42,16 @@
         <artifactId>org.osgi.core</artifactId>
         <license>Apache License, Version 2.0</license>
     </artifact>
+    <artifact>
+        <groupId>org.bouncycastle</groupId>
+        <artifactId>bcprov-jdk15on</artifactId>
+        <license>Bouncy Castle/MIT License</license>
+    </artifact>
+    <artifact>
+        <groupId>io.jsonwebtoken</groupId>
+        <artifactId>jjwt</artifactId>
```

Comment:
you can probably remove this now that you are using `jjwt-api` the license issue looks like it was fixed